.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,49 @@
+name: Cloud Formation Template
+
+on:
+  push: {branches: master}
+  pull_request: {branches: main} 
+  
+jobs:
+  cfn-lint-chk:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Cloud Formation Linter with Latest Version
+        uses: scottbrenner/cfn-lint-action@v2
+      - name: Cloud Formation Version & Verification - Ubuntu.
+        run: |
+          cfn-lint ./templates/nginx-plus-app-protect-ubnt1804-dev.template.yaml --ignore-checks W
+          cfn-lint --version
```

Comment:
This step seems redundant. Separate workflow is going to check the format in advance: https://github.com/f5devcentral/aws-waf-solution-template/pull/13

Reply:
modularized

Reply:
created the separate workflow